### PR TITLE
Addressing cran comments

### PR DESCRIPTION
**Pull request by heemanshusuri**

### CODE REVIEW

File: cran-comments.md [hunk 1]
```diff
@@ -8,3 +8,6 @@
 
 ## R CMD check results
 0 errors | 0 warnings | 0 notes
+
+## Authors and copyright holders
```
Comment (ninghu) [inline cran-comments.md]: Add a “Resubmission” section at the top of cran-comments.md. This should clearly identify that the package is a resubmission, and list the changes that you made. Example:

## Resubmission
This is a resubmission. In this version I have:

* Converted the DESCRIPTION title to title case.

* More clearly identified the copyright holders in the DESCRIPTION
  and LICENSE files.

Comment (ninghu) [reply cran-comments.md]: Check out the "**On failure**" section
http://r-pkgs.had.co.nz/release.html

File: DESCRIPTION [hunk 2]
```diff
@@ -1,17 +1,25 @@
 Package: azuremlsdk
 Type: Package
-Title: R Interface to the Azure Machine Learning Software Development Kit ('SDK')
+Title: Interface to the Azure Machine Learning Software Development Kit ('SDK')
 Version: 0.5.6
 Authors@R: c(
+    person("Heemanshu", "Suri", email = "Heemanshu.Suri@microsoft.com", role = c("cre", "aut")),
     person("Himanshu", "Chandola", email = "Himanshu.Chandola@microsoft.com", role = c("aut")),
```
Comment (ninghu) [inline DESCRIPTION]: Let's update Himanshu's role to ctb

File: DESCRIPTION [hunk 3]
```diff
@@ -1,17 +1,25 @@
...
+    person(family = "Google Inc.", role = c("cph"), comment = "Examples and Tutorials"),
+    person(family = "RStudio Inc.", role = c("cph"), comment = "Examples and Tutorials")
     )
 URL: https://github.com/azure/azureml-sdk-for-r
 BugReports: https://github.com/azure/azureml-sdk-for-r/issues
-Description: An R interface to the Azure Machine Learning service to build and run machine learning workloads.
+Description: Interface to the Azure Machine Learning Software Development Kit
+    ('SDK'). Data scientists can use the 'SDK' to train, deploy, automate, and
+    manage machine learning models on the Azure Machine Learning service. To
+    learn more about Azure Machine Learning visit the website
```
Comment (mx-iao) [inline DESCRIPTION]: can you add a colon after website, e.g. "website:"

File: cran-comments.md (hunk 1 above)
Comment (mx-iao) [reply cran-comments.md]: I'll add the changes to cran-comments.md in a separate PR

Comment (mx-iao) [reply cran-comments.md]: @heemanshusuri you can remove your changes to this file

Comment (heemanshusuri) [reply cran-comments.md]: okay, sounds good.

**Comment by mx-iao** [comment]
cran-comments.md PR: #168

**Pull request merged**